locale/cy/LC_MESSAGES/django.po

```diff
@@ -13,7 +13,7 @@ msgid ""
 msgstr ""
```

Comment:
I ran makemessages and a bunch of other stuff unrelated to this PR happened
I guess changes have been made to the templates and merged without updating the translations

Slight tangent: I think we could do with a CI check to fail the build if you have changed text but not updated the PO files. I'm not going to shave this yak right this second in this PR though.